src/pages/rayon.md

```diff
@@ -0,0 +1,46 @@
+# Rayon
+[![rayon-badge]][rayon]
+
+## Calculate difference of value 1 from each element of an array
```

Comment:
Let's state this so that it is closer to a question a Rust user might ask.

What we ultimately want is for users to come to the cookbook with a question, scan the examples for a use-case statement that sounds like theirs, and see an example of basically what they need to do.

In this case the title of the example is missing some important information, like that it uses parallelism, and that's mutating. Can you come up with a more evocative title?